package.json

```diff
@@ -10,7 +10,7 @@
     "premocha": "npm run lib",
     "mocha": "mocha tests/*.ts tests/**/*.ts --require ts-node/register",
     "postmocha": "rm -rf tests/**/*.js",
-    "test": "npm run lint && npm run mocha && npm run doctest",
+    "test": "npm run lint && npm run mocha",
```

Comment:
Curious, did you mean to include the removal of running doctest as part of this PR, or was it just a local convenience that got committed?